terraform/deployments/govuk-test/main.tf

```diff
@@ -42,18 +42,29 @@ data "terraform_remote_state" "infra_security_groups" {
   }
 }
 
+locals {
+  internal_domain_name  = "${terraform.workspace == "default" ? var.internal_domain_name : "${terraform.workspace}.${var.internal_domain_name}"}" #plouf.test.govuk-internal.digital
+  public_lb_subdomain   = "${terraform.workspace == "default" ? var.govuk_environment : "${terraform.workspace}.${var.govuk_environment}"}"       #plouf.test
+  public_lb_domain_name = "${local.public_lb_subdomain}.${var.public_domain}"                                                                     #plouf.test.govuk.digital
+  mesh_domain           = "${var.mesh_subdomain}.${local.internal_domain_name}"                                                                   #mesh.plouf.test.govuk-internal.digital
+  mesh_name             = "${terraform.workspace == "default" ? var.mesh_name : "${var.mesh_name}-${terraform.workspace}"}"                       #govuk-plouf
+  ecs_cluster_name      = "${terraform.workspace == "default" ? var.ecs_cluster_name : "${var.ecs_cluster_name}-${terraform.workspace}"}"         #ecs-plouf
```

Comment:
The comments are a bit confusing here since they use "govuk", "plouf", and "ecs".

The "plouf" references throughout might be confusing too, but we can change those to `ecs` or remove them in another commit.